src/client/Client.cc

```diff
@@ -2348,6 +2348,13 @@ void Client::handle_client_session(const MConstRef<MClientSession>& m)
   switch (m->get_op()) {
   case CEPH_SESSION_OPEN:
     {
+      if(mdsmap->test_flag(CEPH_MDSMAP_REJECT_CLIENT_SESSION)) {
```

Comment:
I believe this code should go in `Client::mount` after acquiring the MDSMap.

Reply:
> I believe this code should go in Client::mount after acquiring the MDSMap.

So that should be after the if condition and while loop where we check for MDSMap to be available, like https://github.com/ceph/ceph/pull/48720/commits/81ed14e39e7c631941b91d567063729d4b9dc5c8#diff-7a3052fe46aebfed0382c9d0bb9880ea1328add824e0b10c5d551ddfee282cd1R6375-R6379 ?

Reply:
Yes.